### PR TITLE
buffs the ripley

### PR DESCRIPTION
**Pull request by Iamgoofball**
nobody ever builds this shit so let's make it better

1. doubled move speed
2. immune to lava, walk on lava with it
3. doubled ore storage
4. light up wider areas with it
5. made it better armored against lavaland mobs by default but axed its laser/energy resist, could make it useful against revs

### CODE REVIEW

**Comment by Qustinnus** [comment]
the doubled walk speed seems a bit heavy but I havn't seen it in action so I won't judge.

the immune to lava is questionable though. Maybe make it a module for the ripley instead?

**Comment by ninjanomnom** [comment]
the firefighter already is immune iirc

**Comment by XDTM** [comment]
Wasn't the ore storage infinite if you use a box?

**Comment by Iamgoofball** [comment]
yeah but nobody actually knows that, because as stated, no one uses ripleys

**Comment by GunHog** [comment]
> doubled move speed

You have done this incorrectly; It will have no effect. Check the pressure based below it.
>immune to lava, walk on lava with it

Firefighter Ripley. Doing this will remove the incentive to build it instead. Some miners requesting a mech flat out refuse the standard Ripley and demand the upgrade. 

>doubled ore storage

I always thought the was common knowledge; We should inform the player of this instead. Besides, this value is still less than a standard ore satchel.

>light up wider areas with it

Good change.

>made it better armored against lavaland mobs by default but axed its laser/energy resist, could make it useful against revs

How does this stack up with the hide upgrades?

You are wrong when you say "no one uses ripleys". 
196 normal Ripley mechs were made last month. https://atlantaned.space/newSS13tools/stats/monthlyStats.php?year=2017&month=10&stat=mecha_ripley_created

**283** Firefighters were made last month. https://atlantaned.space/newSS13tools/stats/monthlyStats.php?year=2017&month=10&stat=mecha_firefighter_created

479 combined Ripley variants. That is a little over 15 per day last month. I do not think your statement lines up at all.

**Comment by PKPenguin321** [comment]
I agree with gunpoint's stuff except for the last thing, I think the armor values are already kind of flimsy/it's already good at fighting lavaland mobs and doesn't need to be nerfed/buff more in those aspects

**Comment by Iamgoofball** [comment]
i suppose the stats dont lie but i feel they could use a buff still since most mining mains swear off ever touching ripleys

**Comment by XDTM** [comment]
i'm pretty sure they got popular with the last speed changes and have been used to fight bosses and speed mine

**Comment by optimumtact** [comment]
the firefighter ripley may as well just get merged back into the ripley

it was always on the edge of existence

**Comment by Iamgoofball** [comment]
todo: merge firefighter into ripley

@WJohn @Adrinus @nienhaus which sprite do you prefer, Mining RIPLEY or Firefighter RIPLEY

**Comment by PKPenguin321** [comment]
Undo armor changes now!!

**Comment by XDTM** [comment]
greyscale ripleys and make them give them the spraycan_paintable component

**Comment by AnonymousNow** [comment]
I'm all for RIPLEY buffs, but making the Firefighter useless in the process is a little pointless.

Also, if you can take requests, make RIPLEYs have snowflake strengths against xenomorphs. High claw resist, hydraulic clamp grabs and squeezes.

**Comment by Repukan** [comment]
Shouldn't be too good in combat or else it just becomes a rush able double health bar.

**Comment by Iamgoofball** [comment]
@ausops

**Comment by Iamgoofball** [comment]
@repukan
Maybe

**Comment by Repukan** [comment]
That ain't no maybe.

If anything I agree with the utility related buffs but it shouldn't be able to transverse lava and it definitely shouldn't gain more armor when we already have Goliath plating.

**Comment by DoctoreMad** [comment]
I'm not sure why we have a difference between Firefighter RIPLEYs and normal RIPLEYs. It just seems silly.

**Comment by Iamgoofball** [comment]
I'm gonna merge them so

**Comment by DoctoreMad** [comment]
Your doubled move speed and doubled ore storage thing doesn't even work so no I doubt you'll be the guy that made the PR to merge it.

**Comment by Iamgoofball** [comment]
You're an idiot so

File: code/game/mecha/working/ripley.dm [hunk 1]
```diff
@@ -2,18 +2,19 @@
...
+	lights_power = 14
 	deflect_chance = 15
-	armor = list(melee = 40, bullet = 20, laser = 10, energy = 20, bomb = 40, bio = 0, rad = 0, fire = 100, acid = 100)
+	armor = list(melee = 80, bullet = 0, laser = 0, energy = 0, bomb = 40, bio = 0, rad = 0, fire = 100, acid = 100)
+	resistance_flags = LAVA_PROOF
 	max_equip = 6
 	wreckage = /obj/structure/mecha_wreckage/ripley
 	var/list/cargo = new
-	var/cargo_capacity = 15
+	var/cargo_capacity = 30
```
Comment (ChangelingRain) [inline code/game/mecha/working/ripley.dm]: this doesn't double ore storage, which is linked to actual ore boxes.

**Comment by Jalleo** [comment]
Walking on lava with no downsides is BAD. Seriously lava is on purpose to prevent weird edgecases. Like say ripleys charging the syndicate base on lavaland!

File: code/game/mecha/working/ripley.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/mecha/working/ripley.dm]: >mecha code

**Comment by Iamgoofball** [comment]
@Jalleo the downside is slower than average movement/only cardinal movement/slow turning/can't attack back

if you get shot on lava, you're probably dead

**Comment by SgtxRootbeer** [comment]
@Jalleo walking on lava is something the firefighter does regardless.
@Repukan so what if its too good in combat, the amount of ripleys is still limited by the amound of plasma you can round up. and it might be nice having people use mecha more often in combat.

**Comment by KorPhaeron** [comment]
https://github.com/tgstation/tgstation/pull/29996

I argued about this a bunch already, removing every downside from the ripley and making miners immune to basically everything on lavaland for a really cheap mech doesnt sound like it will make the game very exciting

**Comment by optimumtact** [comment]
Did you actually check your assumption it makes them immune to "everything" on lavaland?

**Comment by KorPhaeron** [comment]
@optimumtact 

It makes them immune to ash storms

It makes them immune to lava

It gives them essentially unlimited oxygen

It wont take damage from watcher freeze beams

It cant be stunned by goliath tentacles 

It cant be set on fire 

It now has an 80% resist or whatever to melee attacks though double speed means those attacks wont be hitting you anyway

I guess you could still technically walk into gibtonite or into a chasm

**Comment by AnonymousNow** [comment]
Suggestion - if the normal and Firefighter types are being merged, then make it so that walking on lava is gated behind goliath plating. For each bit of plating, X% fire/lava resistance is added (X being the number of goliath plates that can be applied to the RIPLEY).

**Comment by Dorsisdwarf** [comment]
lavaland is already the one place where the ripley is *already* really good

**Comment by Iamgoofball** [comment]
hm, i see the issue

well, the mining speed stuff is a separate PR, good thing I thought ahead

**Pull request closed without merging**